docs/interpreter/cassandra.md

```diff
@@ -443,37 +427,36 @@ Example:
     @prepare[select]=SELECT * FROM spark_demo.albums LIMIT ?
 
     @prepare[select]=SELECT * FROM spark_demo.artists LIMIT ?
-```                                
+```
 
-For the above example, the prepared statement is _SELECT * FROM spark_demo.albums LIMIT ?_. 
-_SELECT * FROM spark_demo.artists LIMIT ?_ is ignored because an entry already exists in the prepared statements map with the key select. 
+For the above example, the prepared statement is `SELECT * FROM spark_demo.albums LIMIT ?`. 
+`SELECT * FROM spark_demo.artists LIMIT ?` is ignored because an entry already exists in the prepared statements map with the key select. 
 
 In the context of **Zeppelin**, a notebook can be scheduled to be executed at regular interval, 
 thus it is necessary to **avoid re-preparing many time the same statement (considered an anti-pattern)**.
-<br/>
-<br/>
+
 #### b. @bind
-<br/>
-Once the statement is prepared (possibly in a separated notebook/paragraph). You can bind values to it: 
+
+Once the statement is prepared ( possibly in a separated notebook/paragraph ). You can bind values to it: 
 
 ```
     @bind[select_first]=10
-```                                
+```
 
-Bound values are not mandatory for the **@bind** statement. However if you provide bound values, they need to comply to some syntax:
+Bound values are not mandatory for the `@bind` statement. However if you provide bound values, they need to comply to some syntax:
 
 * String values should be enclosed between simple quotes ( ‘ )
 * Date values should be enclosed between simple quotes ( ‘ ) and respect the formats:
   1. yyyy-MM-dd HH:MM:ss
   2. yyyy-MM-dd HH:MM:ss.SSS
-* **null** is parsed as-is
-* **boolean** (true|false) are parsed as-is
+* **Null** is parsed as-is.
+* **Boolean** (true|false) is parsed as-is.
 * collection values must follow the **[standard CQL syntax]**:
   * list: [‘list_item1’, ’list_item2’, ...]
   * set: {‘set_item1’, ‘set_item2’, …}
   * map: {‘key1’: ‘val1’, ‘key2’: ‘val2’, …}
-* **tuple** values should be enclosed between parenthesis (see **[Tuple CQL syntax]**): (‘text’, 123, true)
-* **udt** values should be enclosed between brackets (see **[UDT CQL syntax]**): {stree_name: ‘Beverly Hills’, number: 104, zip_code: 90020, state: ‘California’, …}
+* **Tuple** values should be enclosed between parenthesis ( see **[Tuple CQL syntax]** ): (‘text’, 123, true)
```

Comment:
we might want to keep lowered case for null, boolean tuple and udt since the actual types are lowered case? http://docs.datastax.com/en/cql/3.1/cql/cql_reference/tupleType.html